frontend/js/src/user/playlists/Playlists.tsx

```diff
@@ -197,6 +199,29 @@ export default class UserPlaylists extends React.Component<
                     &nbsp;Spotify
                   </button>
                 </li>
+                <li>
+                  <button
+                    type="button"
+                    onClick={() => {
+                      NiceModal.show<JSPFPlaylist | JSPFPlaylist[], any>(
+                        ImportAppleMusicPlaylistModal
+                      ).then((playlist) => {
+                        if (Array.isArray(playlist)) {
+                          playlist.forEach((p: JSPFPlaylist) => {
+                            this.onPlaylistCreated(p);
+                          });
+                        } else {
+                          this.onPlaylistCreated(playlist);
+                        }
+                      });
+                    }}
+                    data-toggle="modal"
+                    data-target="#ImportSpotifyPlaylistModal"
+                  >
+                    <FontAwesomeIcon icon={faMusic} />
```

Comment:
You should use the apple logo here for consistency 
```suggestion
                    <FontAwesomeIcon icon={faApple} />
```